rules/DeadCode/Rector/ClassConst/RemoveUnusedPrivateClassConstantRector.php

```diff
@@ -67,7 +67,9 @@ public function getNodeTypes(): array
      */
     public function refactor(Node $node): ?int
     {
-        if ($this->shouldSkipClassConst($node)) {
+        $scope = $node->getAttribute(AttributeKey::SCOPE);
```

Comment:
why not use `AbstractScopeAwareRector` with `refactorWithScope` ?

Reply:
Good point :)

Reply:
maybe thats helpful: https://github.com/rectorphp/rector-src/pull/4135